cmd/sonobuoy/app/status.go

```diff
@@ -110,12 +142,17 @@ func humanReadableStatus(str string) string {
 	}
 }
 
+func printJSON(w io.Writer, status *aggregation.Status) error {
+	enc := json.NewEncoder(w)
+	return enc.Encode(status)
+}
+
 func printAll(w io.Writer, status *aggregation.Status) error {
-	tw := tabwriter.NewWriter(w, 1, 8, 1, '\t', tabwriter.AlignRight)
```

Comment:
The alignment was ignored since the padding char was `\t`, it also seemed to make it impossible to slightly tweak the output in a clear fashion. Changed this out for space padding.